.ropeproject/config.py

```diff
@@ -0,0 +1,112 @@
+# The default ``config.py``
```

Comment:
It seems that the `.ropeproject` directory must be ignored. This was possibly generated by your editing process?

Reply:
Yes this might have come from a plugin in atom.

Reply:
Note that you can edit your [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) which is better than polluting the `.gitignore` in this repository.